File: src/PreventionAdvisor/Models/Organization.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using Newtonsoft.Json;

namespace PreventionAdvisor.Models
{
    public class Organization
    {
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        [Key]
        public Guid Id { get; set; }
        public String Name { get; set; }
        public String Vat { get; set; }
        public String Website { get; set; }
        public String Phone { get; set; }
        public Address Address { get; set; }
        public Guid UserId { get; set; }
        [JsonIgnore]
        public ICollection<Workplace> Workplaces { get; set; }
        public virtual AppUser User { get; set; }
    }
}

File: src/PreventionAdvisor/wwwroot/app/organizations/organization.ts
import { IAddress } from './../address/address';

export class IOrganization {
    id: string = '00000000-0000-0000-0000-000000000000';
    name: string = '';
    vat: string = '';
    website: string = '';
    phone: string = '';
    address: IAddress = new IAddress();
}